userspace/engine/lua/rule_loader.lua

```diff
@@ -592,52 +942,22 @@ function load_rules(sinsp_lua_parser,
 	    sinsp_rule_utils.check_for_ignored_syscalls_events(filter_ast, 'rule', v['rule'])
 	 end
 
-	 evttypes, syscallnums = sinsp_rule_utils.get_evttypes_syscalls(name, filter_ast, v['condition'], warn_evttypes, verbose)
+	 evttypes, syscallnums = sinsp_rule_utils.get_evttypes_syscalls(name, filter_ast, v['compile_condition'], warn_evttypes, verbose)
```

Comment:
I do think it's good to keep a verbose flag to suppress this message: https://github.com/falcosecurity/falco/blob/add-exceptions-support/userspace/engine/lua/sinsp_rule_utils.lua#L193. It can be useful when you're debugging, but I don't think you'd want to print it by default.